### PR TITLE
Stop nightly timeouts

### PR DESCRIPTION
**Pull request by scopatz**
Just a test PR for now

### CODE REVIEW

**Comment by scopatz** [comment]
No longer just a test! This seems to work

**Comment by laerus** [comment]
thanks @scopatz !! hope baby travis will stop crying now.

File: xonsh/commands_cache.py [hunk 1]
```diff
@@ -126,10 +125,12 @@ def lazy_locate_binary(self, name):
...
-            local_bin = next((fn for fn in possibilities if os.path.isfile(fn)), None)
+            local_bin = next((fn for fn in possibilities if os.path.isfile(fn)),
+                             None)
             if local_bin:
-                return os.path.abspath(local_bin)                              
-        cached = next((cmd for cmd in possibilities if cmd in self._cmds_cache), None)
+                return os.path.abspath(local_bin)
+        cached = next((cmd for cmd in possibilities if cmd in self._cmds_cache),
+                      None)
         if cached:
```
Comment (melund) [inline xonsh/commands_cache.py]: Oh no.  Was I just PEP8'ed here? Can we change the line endings rules so it doesn't complain hard if we go a few characters above 80. That rule is not giving nicer code ...

Comment (laerus) [inline xonsh/commands_cache.py]: totally agree

Comment (scopatz) [inline xonsh/commands_cache.py]: Well, I have just be using PEP8.  Flake8 is set to 180 right now, which is clearly too high.  If we are going to deviate from PEP8, then we need to document how and in what circumstances.

**Comment by laerus** [comment]
Maybe 90?

**Pull request merged**